src/pubsub/ua_pubsub_writer.c

```diff
@@ -134,6 +135,7 @@ UA_StatusCode
 UA_PublishedDataSetConfig_copy(const UA_PublishedDataSetConfig *src,
                                UA_PublishedDataSetConfig *dst) {
     UA_StatusCode res = UA_STATUSCODE_GOOD;
+    dst->sendViaWriterGroupTopic = src->sendViaWriterGroupTopic;
```

Comment:
Non nested fields are copyed by the memcpy afterwards.

Reply:
I removed this.